frontend/src/utils/AxiosReporterApi.ts

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const AxiosReportersApi = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com/",
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+
```

Comment:
You can definitly have multiple instances.

Create one for the back-end (`localhost:5000`)